buildpack/telemetry/metrics.py

```diff
@@ -24,14 +24,43 @@
 # This enables the new stream of metrics coming from micrometer instead
 # of the admin port.
 # https://docs.mendix.com/refguide/metrics#registries-configuration
-METRICS_REGISTRY = [
+PAIDAPPS_METRICS_REGISTRY = [
     {
         "type": "influx",
         "settings": {
             "uri": "http://localhost:8086",
             "db": "mendix",
             "step": "10s",
         },
+        "filters": [
```

Comment:
Can we also add a comment for pointing out the official doc link for setting up this filter: https://docs.mendix.com/refguide/metrics#filters